dev/com.ibm.ws.security.social/src/com/ibm/ws/security/social/internal/OidcLoginConfigImpl.java

```diff
@@ -129,6 +131,9 @@ protected void setRequiredConfigAttributes(Map<String, Object> props) {
 
     @Override
     protected void setOptionalConfigAttributes(Map<String, Object> props) throws SocialLoginException {
+
+        this.useSystemPropertiesForHttpClientConnections = configUtils.getBooleanConfigAttribute(props, KEY_USE_SYSPROPS_FOR_HTTPCLIENT_CONNECTONS, false);
+        this.userInfoEndpoint = configUtils.getConfigAttribute(props, KEY_USERINFO_ENDPOINT);
```

Comment:
Did this `this.userInfoEndpoint =` line slip in here? It doesn't look like it was moved from somewhere else -  it looks like it's a new addition.